cloudprem/logical/static/grafana_values.yml

```diff
@@ -0,0 +1,107 @@
+image:
+  repository: grafana/grafana
+  tag: "7.5.16"
+  pullPolicy: IfNotPresent
+
+
+## Deployment annotations
+# annotations: {}
+
+## Expose the grafana service to be accessed from outside the cluster (LoadBalancer service).
+## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
+## ref: http://kubernetes.io/docs/user-guide/services/
+##
+service:
+  enabled: true
+  type: NodePort
+  port: 80
+  nodePort: 32020
+
+persistence:
+  type: pvc
+  enabled: true
+  accessModes:
+    - ReadWriteOnce
+  size: 10Gi
+  finalizers:
+    - kubernetes.io/pvc-protection
+
+initChownData:
+  ## If false, data ownership will not be reset at startup
+  ## This allows the prometheus-server to be run with an arbitrary user
+  ##
+  enabled: false
+
+# Administrator credentials when not using an existing secret (see below)
+adminUser: dozuki
+# adminPassword: s
```

Comment:
Should this be commented out? If not, should there be a value other than 's'?

Reply:
If you look here you'll see where the Grafana admin password is generated and it's used below during the helm chart deploy. That line should probably just be removed but it's not hurting anything to be there. https://github.com/Dozuki/CloudPrem-Infra/pull/40/files#diff-14998cb297a4ddd24e0d8e69a0e8624ba131d8148378c1b275f72c97783c58abR35

Reply:
I think the thing that threw me off in particular was the `(see below)` because I looked further in the file and didn't see anything in particular about filling in the password.

Reply:
https://github.com/Dozuki/CloudPrem-Infra/pull/40/files#diff-14998cb297a4ddd24e0d8e69a0e8624ba131d8148378c1b275f72c97783c58abR59